packages/web/components/cards/apr-breakdown.tsx

```diff
@@ -61,7 +62,14 @@ export const AprBreakdown: FunctionComponent<
             }
           )}
         >
-          <p>{t("pools.aprBreakdown.total")}</p>
+          {showDisclaimerTooltip ? (
+            <div className="flex items-center gap-1">
+              <p>{t("pools.aprBreakdown.total")}</p>
+              <AprDisclaimerTooltip />
+            </div>
+          ) : (
+            <p>{t("pools.aprBreakdown.total")}</p>
```

Comment:
Thoughts on moving this to a variable? 

```
const aprBreakdownTotal = <p>{t("pools.aprBreakdown.total")}</p>
``` 

```suggestion
            aprBreakdownTotal
```